# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1357157768 18000
#      Wed Jan 02 15:16:08 2013 -0500
# Node ID df1aceb8f0bc6b5b5062907931cc663467f57d93
# Parent  f69530e3600d95aae46d308ec4cf400452f2bd5a
snapshot version 3.7.1

* configure.ac (AC_INIT): Set version to 3.7.1.
(OCTAVE_RELEASE_DATE): Set to 2013-01-02.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,23 +14,23 @@ dnl Copyright (C) 1993-2012 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ### 
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.62])
-AC_INIT([GNU Octave], [3.7.0+], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.7.1], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="48"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
-OCTAVE_RELEASE_DATE="2012-02-22"
+OCTAVE_RELEASE_DATE="2013-01-02"
 OCTAVE_COPYRIGHT="Copyright (C) 2012 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
 AC_SUBST(OCTAVE_API_VERSION_NUMBER)
 AC_SUBST(OCTAVE_API_VERSION)
 AC_SUBST(OCTAVE_RELEASE_DATE)
 AC_SUBST(OCTAVE_COPYRIGHT)
 
 dnl FIXME: We should auto-insert the Mercurial changeset ID into the
